CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ## [Unreleased]
 ### Fixed
 - missing taxes on child products during item export
+- fix redirect and images in order history view
```

Comment:
please be more descriptive which redirect and images are fixed here

Reply:
Remove the fix, as it is mention by ### Fixed